src/install/unix.rst

```diff
@@ -222,6 +222,8 @@ From here you should verify your installation by pointing your web browser to::
 
     http://localhost:5984/_utils/verify_install.html
 
+Finally, open "Configure" page and configure a cluster. After this all warning messages in a log will go away.
```

Comment:
Could you add a link to here http://docs.couchdb.org/en/latest/cluster/setup.html?highlight=setup#the-cluster-setup-wizard for the configuration part